Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -560,6 +560,7 @@ WebPageProxy::WebPageProxy(PageClient& pageClient, WebProcessProxy& process, Ref
 
 #if ENABLE(REMOTE_INSPECTOR)
     m_inspectorDebuggable->setInspectable(JSRemoteInspectorGetInspectionEnabledByDefault());
+    m_inspectorDebuggable->setPresentingApplicationPID(m_configuration->processPool()->configuration().presentingApplicationPID());
```

Comment:
it seems like most other things access this via `m_process->processPool()`.  should we do the same?